### PR TITLE
Ipam-autopilot | Fix routing domain update

### PR DESCRIPTION
**Pull request by oskisk**
Adds error handling for the routing domain update endpoint and fixes the update query of the routing domain.

fixes https://github.com/GoogleCloudPlatform/professional-services/issues/970

### CODE REVIEW

**Review by cgrotz — APPROVED** [review]
LGTM, Thanks @oskisk for contributing. I will merge as soon as the CLA is cleared.

**Comment by oskisk** [comment]
> @oskisk Still waiting on the CLA. We can merge this as soon as the CLA is complete.

Thank you, our legal department has signed the CLA and we are waiting for the approval from google.

**Comment by oskisk** [comment]
> @oskisk Still waiting on the CLA. We can merge this as soon as the CLA is complete.

@agold-rh We are waiting for the CLA request to be granted by Google, is there any way to expedite the process?

**Pull request merged**
